s4e-backend/src/main/java/pl/cyfronet/s4e/service/AppUserService.java

```diff
@@ -24,4 +26,8 @@ public AppUser save(AppUser appUser) throws AppUserCreationException {
             throw new AppUserCreationException(e);
         }
     }
+
+    public Optional<AppUser> findByEmail(String email) {
```

Comment:
should we return from services one type of return, optional or e.g. AppUser/boolean ?